runtime/j9vm/jvm.c

```diff
@@ -1218,73 +1226,148 @@ decodeSetting(const char* key, const char* value, VersionSetting* settings, IDAT
 }
 
 /**
- * Loads the classlib.properties file and initialize the bootstrap
- * classpath based on data found there.
- * @param vm
- * @param cursor
- * @return TRUE on success, FALSE otherwise.
+ * Get Java version info from 'classlib.properties' file
+ * Attempt loading 'classlib.properties' file, if found, 'shape' & 'version' values are retrieved,
+ * and decoded as J2SE_xx and J2SE_SHAPE_xx accordingly. 
+ * J2SE_xx | J2SE_SHAPE_xx will be returned.
```

Comment:
This is a better description for @return, but all the behavior when classlib.properties isn't found.